.ci/containers/github-differ/generate_comment.sh

```diff
@@ -144,6 +146,19 @@ if git clone -b $NEW_BRANCH $TFV_SCRATCH_PATH $TFV_LOCAL_PATH; then
     popd
 fi
 
+# TGC
```

Comment:
It looks like this is actually not necessary to add, since the old TGC handling in this script is still in place and apparently still works! 😮  https://github.com/GoogleCloudPlatform/magic-modules/blob/af37bb70c6a9201ed3e754df655ce4c08fb366e1/.ci/containers/github-differ/generate_comment.sh#L123

(I can tell because I see a TF Conversion line in the diff comment, but this section is using `TF Validator` still.) https://github.com/GoogleCloudPlatform/magic-modules/pull/7511#issuecomment-1488939140

Reply:
Removed.